Misc/NEWS.d/next/Library/2020-05-25-12-42-36.bpo-17258.lf2554.rst

```diff
@@ -0,0 +1 @@
+:mod:`multiprocessing` supports stronger HMAC algorithms
```

Comment:
Explain what they're used for as it is an internal implementation detail that they get used.